sdk/src/types/block/rand/block.rs

```diff
@@ -22,8 +24,8 @@ pub fn rand_block_ids(len: usize) -> Vec<BlockId> {
 }
 
 /// Generates a random block with given parents.
-pub fn rand_block_with_parents(parents: Parents) -> Block {
-    BlockBuilder::new(parents)
```

Comment:
probably the builder should have `.with_strong_parents`, `.with_weak_parents`, ... methods?

Reply:
As said on Slack, this is not the purpose of this PR. I will add them when I actually update the Block model.

Reply:
maybe also `Parents` should be come an enum now:
```rust
enum Parents {
    Strong(_),
    Weak(_),
    ShallowLike(_)
}
```

Reply:
I don't think so, these are 3 distinct fields so there is no need for an enum, would make everything much more complex for no real reason

Reply:
okay